### PR TITLE
Document flat draw rule for point styles

### PR DESCRIPTION
**Pull request by tallytalwar**

### CODE REVIEW

File: pages/draw.md [hunk 1]
```diff
@@ -111,6 +111,13 @@ text:
     anchor: bottom-left # the label will use `align: right` by default
 ```
 
+####`flat` 
+[[ES-only](https://github.com/tangrams/tangram-es)]
```
Comment (nvkelso) [inline pages/draw.md:115]: Is `flat` a standard 3d term? (I've never heard of it before.)

Comment (matteblair) [reply pages/draw.md:115]: It's used with the same meaning in Google Maps: https://developers.google.com/maps/documentation/android-api/marker#flatten_a_marker

**Comment by meetar** [comment]
@tallytalwar What does "will not be affected by viewport" mean?

**Comment by tallytalwar** [comment]
screen shot 2017-02-06 at 12 55 37 pm

@meetar by that I mean view orientation, since in ES we can rotate and tilt the view. Refer the image above, `flat` is set to `true` on the left and `false` on the right.

**Comment by meetar** [comment]
… I can't actually tell what's different here apart from the perspective
change and label placement 😅

Maybe it would make more sense during a camera move?

**Comment by tallytalwar** [comment]
Look at the arrows pointing not along the road on the right but almost perpendicular to the road!

**Comment by meetar** [comment]
Thanks @tallytalwar!

**Comment by meetar** [comment]
One last thing: default should be 'false', correct?

**Comment by tallytalwar** [comment]
Yup, default is `false`.

**Pull request merged**